### ISSUE TITLE
role makes homedirs more permissive

### ISSUE THREAD

**Issue by jamescassell** [MindPointGroup/RHEL7-STIG#122]
```
TASK [R7S : MEDIUM | RHEL-07-020630 | PATCH | All local interactive user home directories must have mode 0750 or less permissive.] **
--- before
+++ after
@@ -1,4 +1,4 @@
 {
-    "mode": "0700",
+    "mode": "0750",
     "path": "/home/admin"
 }
```

we should only downgrade the permissions, not upgrade.

**Comment by mikerenfro** [comment]
PR #128 should fix all this now, once it's done with Travis.

**Comment by shepdelacreme** [comment]
Fixed in #128
